DEVEL.md

```diff
@@ -0,0 +1,78 @@
+# Developing the Maistra Istio Operator
+
+## Building the binary and the container image
+
+To build the Operator container image, run:
+```bash
+IMAGE=docker.io/<your username>/istio-operator:latest tmp/build/docker_build.sh
+```
+
+This builds the binary, downloads and patches the Istio Helm charts, and finally builds the container image. The build script doesn't push the image to the container registry, so you'll need to push it manually:
+```bash
+docker push docker.io/<your username>/istio-operator:latest
+```
+
+## Running the operator locally
+
+During development, you can run Istio Operator locally on your dev machine and have it reconcile objects in a local or remote OpenShift cluster. You can run the Operator from within your IDE and even use a debugger.
+
+To run the Operator locally, you need to set the following environment variables:
+
+- `KUBECONFIG=<path to your kubectl config file>`
+- `POD_NAME=istio-operator`
+- `POD_NAMESPACE=istio-operator`
+- `ISTIO_CNI_IMAGE=some-registry/istio-cni-image:latest`
+- `ISTIO_CNI_IMAGE_PULL_SECRET=my-pull-secret` (required if the registry hosting the CNI image is not public; secret must be in operator's namespace as defined above)
+
+Make sure you run `tmp/build/docker_build.sh` before running the Operator to generate the Helm charts in `tmp/_output/helm`.
```

Comment:
What about

Make sure you run `tmp/build/docker_build.sh` before running the Operator to generate the required resources in `tmp/_output/resources`.

Reply:
done.